openedx/openedx-platform
# fix: Update course preview redirect

## Description

This PR changes the preview URL redirect from Studio. Before when a user clicked on the course preview button to check unpublished changes, the preview in the New Experience would show the most recently published version of the course. The unpublished changes that the user was hoping to view are no present. At the moment users have been publishing the content to check it then having to republish the old version if they did not like the changes. As a short-term fix, this PR redirects to the Legacy experience where the preview option previously worked. Now the user can see the unpublished changes. A long-term fix is still being explored. This change will impact the Course Author.

**BEFORE**
![Animated GIF-downsized_large (3)](https://user-images.githubusercontent.com/42981026/121414813-12281b00-c935-11eb-9eae-17a25f53793e.gif)

**AFTER**
![Animated GIF-downsized_large (4)](https://user-images.githubusercontent.com/42981026/121415050-51ef0280-c935-11eb-8e7d-bb0c2899f56f.gif)

## Supporting information

Jira issue: [TNL-8236](https://openedx.atlassian.net/browse/TNL-8236)

## Testing instructions

- In Studio view, add a new block.
- Check that your changes are labeled "Draft (unpublished changes)".
- Click on the "Preview" button.
- Check that you are in the Legacy view.
- Scroll to find your new block.

## Deadline

None

## Other information

This is a short-term fix but will require a long-term fix.
